src/rdb/rdb_raft.c

```diff
@@ -2521,6 +2521,18 @@ rdb_raft_resign(struct rdb *db, uint64_t term)
 	D_ASSERTF(rc == 0, ""DF_RC"\n", DP_RC(rc));
 }
 
+/* TODO: update raft.h in daos-stack/raft repo */
+extern int raft_election_start(raft_server_t *me_);
```

Comment:
(style)  externs should be avoided in .c files

Reply:
Will remove this and update the raft submodule once the daos-stack/raft repository PR lands https://github.com/daos-stack/raft/pull/36